maintainer/maintainer.go

```diff
@@ -412,6 +418,9 @@ func (m *Maintainer) onMessage(msg *messaging.TargetMessage) {
 	case messaging.TypeCheckpointTsMessage:
 		req := msg.Message[0].(*heartbeatpb.CheckpointTsMessage)
 		m.onCheckpointTsPersisted(req)
+	case messaging.TypeRedoTsMessage:
+		req := msg.Message[0].(*heartbeatpb.RedoTsMessage)
+		m.onRedoTsPersisted(msg.From, req)
```

Comment:
It is not a particularly good idea to send redoTsMessage to everyone every time a dispatcher reports it. In the case of many nodes, a new one may arrive after the new one is sent. I think it can be made into a peroid task to avoid a large number of useless messages.